tests/snippets/testutils.py

```diff
@@ -20,7 +20,17 @@ def assert_raises(exc_type, expr, msg=None):
         assert False, failmsg
 
 
-class assertRaises:
+def assertRaises(expected, *args, **kw):
```

Comment:
What does this function do? We already have another function` assert_raises`, is this what you were looking for? Note that this function might be tricky since it now becomes very versatile.

Reply:
About the tricky interface, it is just made for interface compatibility with unittest.assertRaises. Because the basic implementation of assertRaises was compatible with it, i thought it is intended to be compatible.